connectivity/model/src/main/java/org/eclipse/ditto/connectivity/model/mqtt/MessageExpiryInterval.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.connectivity.model.mqtt;
+
+import java.util.Objects;
+import java.util.OptionalLong;
+
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Representation of the MQTT 5 message expiry interval.
+ * The minimum seconds is {@value MIN_INTERVAL_SECONDS}.
+ * The maximum seconds is {@value MAX_INTERVAL_SECONDS}.
+ */
+@Immutable
+public class MessageExpiryInterval {
+
+    public static final long MIN_INTERVAL_SECONDS = 1L;
+
+    public static final long MAX_INTERVAL_SECONDS = 4_294_967_295L;
```

Comment:
Those are ~136 years, is that correct?
Is this max value specified?

Seems a little too much to be reasonable.

Reply:
[Specification](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901112) says that the value is 4-byte integer value. Nothing about maximum value.
HiveMQ doc [says](https://github.com/hivemq/hivemq-community-edition/wiki/MQTT-Specific-Configuration#mqtt-specific-configuration-options) that maximum value for message expiry is 4.294.967.296. This seems strange to me, cause this takes more than 4 bytes.
So I took maximum 4-byte unsinged value.